src/Compilers/CSharp/Portable/Binder/UsingStatementBinder.cs

```diff
@@ -168,11 +170,12 @@ internal static BoundStatement BindUsingStatementOrDeclarationFromParts(SyntaxNo
                     hasErrors);
             }
 
-            // initializes iDisposableConversion, awaitableTypeOpt and disposeMethodOpt
-            bool populateDisposableConversionOrDisposeMethod(bool fromExpression)
+            bool populateDisposableConversionOrDisposeMethod(bool fromExpression, out Conversion iDisposableConversion, out MethodSymbol disposeMethodOpt, out TypeSymbol awaitableTypeOpt)
```

Comment:
>out Conversion iDisposableConversion, out MethodSymbol disposeMethodOpt, out TypeSymbol awaitableTypeOpt [](start = 82, length = 104)

Updated local function signature to make explicit the initialization of locals in outer method.